cleanlab_studio/studio/studio.py

```diff
@@ -383,3 +386,30 @@ def poll_cleanset_status(self, cleanset_id: str, timeout: Optional[int] = None)
 
         except (TimeoutError, CleansetError):
             return False
+
+    def get_autofix_defaults(self, project_id: str) -> dict:
+        """
+        Returns the default parameters for autofix.
+        Args:
+            project_id: ID of project.
+
+        Returns:
+            A dictionary containing number of rows to drop for each issue type.
+        """
+        cleanset_id = api.get_latest_cleanset_id(self._api_key, project_id)
+        cleaned_df = self.download_cleanlab_columns(cleanset_id)
+        return _get_autofix_defaults(cleaned_df)
+
+    def autofix_dataset(self, project_id: str, params: dict = None) -> pd.DataFrame:
+        """
+        Args:
+            project_id: ID of project.
+            params: Default parameter dictionary showing number of rows to drop for each issue type.
```

Comment:
not just drop right, arent we re-labeling some of the label issues too?

Reply:
give an example in the docstring here of what this dict might look like.

Reply:
Yes confidence threshold is used as well, updated Docstring with example: 
https://github.com/cleanlab/cleanlab-studio/blob/d67bbc37c052929a2fbd134c57dcd62f2e24bb27/cleanlab_studio/studio/studio.py#L400